spec/controllers/overrides/registrations_controller_spec.rb

```diff
@@ -47,34 +55,38 @@
   end
 
   describe '#update' do
-    # We no longer override the update method, but we want to confirm that email
-    # change confirmation emails are being sent.
     let(:user_update_params) do
       {
         user: {
-          id: 1,
-          email: 'jane2@doe.com',
+          email: 'jack2@doe.com',
```

Comment:
wondering what the id is for jack2doe and how the update finds the id?/user?

Reply:
if you look at the [update method on the base class](https://github.com/lynndylanhurley/devise_token_auth/blob/master/app/controllers/devise_token_auth/registrations_controller.rb#L69) it isn't querying by id, it's applying update changes to the existing @resource, which we assigned to the user in the `before` block.